### DISCUSSION TITLE
Timeout on "Preparing build machine" for Mac mini M1 builds

### DISCUSSION THREAD

**Discussion by casimir** [codemagic-ci-cd/codemagic-docs#2230] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

iOS (Swift, Objective-C)

### Steps to reproduce

Start a build with `xcode: 15` in the workflow configuration. The behaviour is the same for all available XCode versions.

### Expected results

A Max mini M1 machine is attributed and the workflow is executed.

### Actual results

The build is stuck on "Preparing build machine" step.
<img width="1233" alt="image" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/879004/bb3d30ba-5eb3-4e4a-a005-c190c6b124c8">

### Build id (optional)

656707a0fbd983db1b1404ac

**Comment by himesh-cm** [comment]
Hi @casimir, services are working normally, are you still facing this issue?

**Reply by casimir** [reply]
Still facing the issue, I just tested (build `65699c31f1a80d6965c3258e`). Also I forgot to mention that I encounter this issue consistently since 8 days now.

**Reply by himesh-cm** [reply, marked as answer]
Hi @casimir,

Please check this and ensure you are not using env vars created in a different team.


`ValueError: - encrypted variables work only with builds in the same team they were created with.`

**Comment by MatthieuFereyre** [comment]
Hi, I'm facing the same issue on Mac mini M1. Preparing machine is in timeout.

**Reply by himesh-cm** [reply]
@MatthieuFereyre, can you please share your build id so I can further investigate the issue?

**Reply by MatthieuFereyre** [reply]
Hello, here is the last build id that failed due to timeout : 656d8fb319f3013c85abaced

**Reply by MatthieuFereyre** [reply]
I have no more timeout on builds. I just updated my codemagic.yaml file content with the exemple (for me it was ionic-capacitor) in https://github.com/codemagic-ci-cd/codemagic-sample-projects/tree/main
The syntax has changed a lot (my file was 5 years old or something).

**Comment by himesh-cm** [comment]
While I’m closing our discussion for now, if you have any other questions or suggestions, feel free to reply back or start a new discussion anytime. Have a nice day!
